wodles/azure/azure-logs.py

```diff
@@ -77,9 +78,11 @@ def get_script_arguments():
 
     # Log Analytics arguments #
     parser.add_argument("--la_id", metavar='ID', type=str, required=False,
-                        help="Application ID for Log Analytics authentication.")
+                        help="Application ID for Log Analytics authentication. This option was deprecated in 4.4. "
```

Comment:
Same suggestion as in AWS module. Some parts of the help messages are common and can be improved using a constant in order to maintain the readability of code.

Reply:
Done